cps/main.c

```diff
@@ -1080,6 +1082,7 @@ add_bgp_filters(struct gatekeeper_if *iface, uint16_t tcp_port_bgp,
 					iface->name);
 				return ret;
 			}
+			*rx_method = RX_METHOD_NIC;
```

Comment:
I know that this line is going to be removed by a patch later, but we should be consistent and use `|=` instead of `=`. There are other locations in this patch with the same inconsistency.